codespell_lib/data/dictionary.txt

```diff
@@ -10976,6 +10976,7 @@ capabilitiy->capability
 capabillities->capabilities
 capabillity->capability
 capabilties->capabilities
+capabiltiies->capabilities
```

Comment:
I am OK to merge as is, but I would suggest also adding:
```
capabiltiy->capability
```

Reply:
This is a great addition; I have also added this spelling variant. Thank you!